src/current/cockroachcloud/network-authorization.md

```diff
@@ -1,45 +1,52 @@
 
 {{site.data.alerts.callout_info}}
+
 During [limited access](/docs/{{site.versions["stable"]}}/cockroachdb-feature-availability.html), neither Azure Private Link nor private clusters are available for {{ site.data.products.dedicated }} clusters on Azure. Refer to [{{ site.data.products.dedicated }} on Azure](cockroachdb-dedicated-on-azure.html).
 {{site.data.alerts.end}}
 
-## IP allowlisting
+## Cluster default network configuration
 
-Authorize your application server’s network and your local machine’s network by [adding their public IP addresses (in the CIDR format) to the {{ site.data.products.dedicated }} cluster's allowlist](connect-to-your-cluster.html#authorize-your-network). If you change your location, you will need to authorize the new location’s network, else the connection from that network will be rejected.
+{{ site.data.products.dedicated }} and Serverless clusters differ in their default network configuration:
 
-- In a development environment, you need to authorize your application server’s network and your local machine’s network. If you change your location, you need to authorize the new location’s network, or else the connection from that network will be rejected.
-- In a production environment, you need to authorize your application server’s network.
+- On creation, a Serverless cluster is open to all traffic as it is created with a 0.0.0.0/0 IP allowlist entry.
+- On creation, a Dedicated cluster is "locked down" and has no access until an authorized network is created.
 
-{{site.data.alerts.callout_info}}
-While developing and testing your application, you may add `0.0.0.0/0` to the allowlist, which allows all networks. However, before moving into production, make sure you delete the `0.0.0.0/0` network.
-{{site.data.alerts.end}}
+{{ site.data.products.db }} clusters can only accept SQL connections from [allowed IP addresses](#ip-allowlisting).
+
+## IP allowlisting
+
+Authorized network access can be managed from the {{ site.data.products.db }} console Network Authorization page at:
+
+`https://cockroachlabs.cloud/cluster/{ your cluster UUID}/networking`
 
 You can add up to 20 IP addresses to your allowlist. If your application servers’ IP addresses are not static, or you want to limit your cluster's exposure to the public network, you can connect to your {{ site.data.products.dedicated }} clusters using VPC Peering or AWS PrivateLink instead.
 
-{{site.data.alerts.callout_success}}
-During [limited access](/docs/{{site.versions["stable"]}}/cockroachdb-feature-availability.html), neither Azure Private Link nor private clusters are available for {{ site.data.products.dedicated }} clusters on Azure. Refer to [{{ site.data.products.dedicated }} on Azure](cockroachdb-dedicated-on-azure.html).
-{{site.data.alerts.end}}
+Refer to:
+
+- [Connect to a {{ site.data.products.serverless }} Cluster: Authorize your network](connect-to-a-serverless-cluster.html#authorize-your-network)
```

Comment:
Can we make sure that the flow and organization of information is such that the core info is all within the specific new page and we're just linking to that from other places?